admiral/pkg/controller/common/config.go

```diff
@@ -438,10 +438,18 @@ func GetIngressVSExportToNamespace() []string {
 	return wrapper.params.IngressVSExportToNamespaces
 }
 
-func IsVSBasedRoutingEnabled() bool {
+func DoVSRoutingForCluster(cluster string) bool {
 	wrapper.RLock()
 	defer wrapper.RUnlock()
-	return wrapper.params.EnableVSRouting
+	if !wrapper.params.EnableVSRouting {
+		return false
+	}
+	for _, c := range wrapper.params.VSRoutingEnabledClusters {
```

Comment:
If `EnableVSRouting` is `true` and `VSRoutingEnabledClusters` is "[]", does that mean no clusters should get the VS? 

What is the correct way to enable it across the board without adding clusters to the list?

Reply:
+1

Reply:
@adilfulara , I'll be adding a check for `*`. If that exists in the cluster list, then it will be enabled for all clusters.